db_migration/alembic/versions/20220104_174013_cdad7efc63be_initialize_database.py

```diff
@@ -24,11 +24,11 @@ def upgrade():
         sa.Column("file_id", sa.String(), nullable=False),
         sa.Column("md5_checksum", sa.String(), nullable=False),
         sa.Column("size", sa.Integer(), nullable=True),
-        sa.Column("study_id", sa.String(), nullable=False),
+        sa.Column("grouping_label", sa.String(), nullable=False),
```

Comment:
alembic version files should never be changed (this defeats the purpose of having version file in the first place)
Was this intentional?